liquid_node/backup.py

```diff
@@ -38,6 +38,13 @@ def backup(*args):
         if config.is_app_enabled('dokuwiki'):
             backup_files(dest / 'dokuwiki.tgz', '/bitnami/dokuwiki', [], 'dokuwiki:php')
 
+        if config.is_app_enabled('hypothesis'):
+            collection_dir = dest / "hypothesis"
```

Comment:
`collection_dir` makes you think there's a hoover collection is involved, but this is not the case. Please use another name for this variable.